app/scripts/services/builds.js

```diff
@@ -88,8 +88,10 @@ angular.module("openshiftConsole")
         namespace: build.metadata.namespace
       };
       var canceledBuild = angular.copy(build);
+      var rgv = APIService.objectToResourceGroupVersion(canceledBuild);
       canceledBuild.status.cancelled = true;
-      return DataService.update(buildsVersion, canceledBuild.metadata.name, canceledBuild, context).then(function() {
```

Comment:
Should be able to remove `buildsVersion` var now

Reply:
Not sure why jshint didn't catch that when I did my build, just removed.